src/OnlineRetailPortal.Core/Translators/UpdateProductTranslator.cs

```diff
@@ -7,30 +7,63 @@ namespace OnlineRetailPortal.Core
 {
     public static class UpdateProductTranslator
     {
-        public static ProductEntity ToStoreEntity(this Product product)
+        public static ProductEntity ToStoreEntity(this Product updateProduct)
```

Comment:
product is renamed to updateProduct, how can updateProduct again be translated into updateProductStoreRequest ? please rename it

Reply:
done